server/chat.ts

```diff
@@ -97,6 +97,15 @@ export type PunishmentFilter = (user: User | ID, punishment: Punishment) => void
 export type LoginFilter = (user: User, oldUser: User | null, userType: string) => void;
 export type HostFilter = (host: string, user: User, connection: Connection, hostType: string) => void;
 
+export interface TRStrings {
+	[k: string]: string;
+}
+
+export interface Translations {
+	name?: string;
+	strings: TRStrings;
+}
```

Comment:
```suggestion
export interface Translations {
	name?: string;
	strings: {[english: string]: string};
}
```

It doesn't look like you actually use the `TRStrings` type anywhere.

Reply:
Hm, I think that was just a leftover from before.